src/edu/ucsb/nceas/metacat/admin/PropertiesAdmin.java

```diff
@@ -251,7 +341,7 @@ public void configureProperties(HttpServletRequest request,
                     SolrVersionChecker checker = new SolrVersionChecker();
                     isOldVersion = checker.isVersion_3_4(solrHomePath);
                 } catch (Exception e) {
-                    logMetacat.warn("PropertiesAdmin.configureProperties - we can't determine if the given directory is a old version of solr  since "+e.getMessage()+". But we consider it is not an old version.");
+                    logMetacat.warn("PropertiesAdmin.configureProperties - we can't determine if the given solr home directory is a old version of solr  since "+e.getMessage()+". But we consider it is not an old version.");
```

Comment:
Please wrap at 100 char column.

Suggested rewording:

we can't determine if the given solr home directory ~~is a~~ `contains an` old version of solr`,` since "` `+` `e.getMessage()` `+` `". ~~But we consider~~ `We will assume` it is not an old version.